src/black/comments.py

```diff
@@ -399,9 +399,8 @@ def _contains_fmt_skip_comment(comment_line: str, mode: Mode) -> bool:
             ],
             *[
                 _COMMENT_PREFIX + comment.strip()
-                for comment in comment_line.strip(_COMMENT_PREFIX).split(
-                    _COMMENT_LIST_SEPARATOR
-                )
+                for comment in comment_line.strip(_COMMENT_PREFIX)
```

Comment:
This feels worse than the previous formatting.

Reply:
I also don't like the new one, I personaly like is: 
```python
                for comment in (
                    comment_line.strip(_COMMENT_PREFIX).split(_COMMENT_LIST_SEPARATOR)
                )
```
anyway, I will revert this change.